src/pages/mypage/page.tsx

```diff
@@ -0,0 +1,28 @@
+// import Sidebar from './component/SideBar';
+import ProfilePage from './profile/page';
+
+export default function MyPage() {
+  return (
+    <div className="mt-24">
+      <div>
+        <h1 className="mb-5 ml-72 text-2xl font-bold">마이페이지</h1>
+        <div className="border-#E7E7E7 w-full border-b"></div>
```

Comment:
```suggestion
        <div className="border-[#E7E7E7] w-full border-b"></div>
```

Arbitrary value는 대괄호로 감싸줘야 하는 걸로 알고 있는데, 요렇게 작성해도 적용이 되는가요? 궁금합니다

Reply:
헉 이건 제가 tailwind를 처음 써봐서 신경쓰지 못한 부분인데 적용은 되더라구요 ! 이후 작업부터는 신경 쓰도록 하겠습니다 !!

Reply:
제가 해봤을 땐 적용이 안되어서... 다시 한번 확인해 주시고 되면 공유 부탁드려요~